### ISSUE TITLE
installation OnvifDeviceManager : Problem in Kubuntu 22.04

### ISSUE THREAD

**Issue by Danieleeffe1** [Quedale/OnvifDeviceManager#28]
good morning

Thanks for making this software, it was very useful to me, but since it was installed I noticed some differences in the system.

I have a Kubuntu 22.04 installation
1) when I start a terminal it does not open unless I move the mouse or click one of its buttons.
2) when the terminal is started and I type a command I do not see a response until I move the mouse or click one of its buttons.
3) the mouse pointer has changed its shape to CROSS.

At first I thought it was just a problem localized to the account that I normally use so I created a new Kubuntu account, but the problem is also present in the new account.

Also

I use a software that requires CONDA/MINICONDA.

when I try to start conda it gives me this error:

import-im6.q16: attempt to perform an operation not allowed by the security policy `PS' @ error/constitute.c/IsCoderAuthorized/426.
/media/daniele/620765BD563684C4/miniconda3/bin/conda: line 6: syntax error near unexpected token "sys.argv"
/media/daniele/620765BD563684C4/miniconda3/bin/conda: line 6: `if len(sys.argv) > 1 and sys.argv[1].startswith('shell.') and sys.path and sys.path[0] == ''

what should I do to restore everything as before?

**Comment by Quedale** [comment]
Good day!


I can assure you that nothing in the system was changed. (Apart mandatory packages on README.md)
I especially designed the autogen.sh script to work entirely within the project folder. (no sudo privilege required)
Required library are statically linked (when possible), they don't actually get installed along the onvifmgr executable.
You can actually start using the executable straight from the build folder without installing it.

I unfortunately didn't get to a fully functional distributable yet.
If you delete the subprojects folder, the installed executable may not work anymore. (TODO before first release)
I didn't get to this part yet, because I want to be sure I'm not accidentally breaking people's system.

if you look at the [Makefile.am](https://github.com/Quedale/OnvifDeviceManager/blob/main/Makefile.am), on `line 130` you can see that only the executable is actually included in the package.
```
	@mkdir onvifmgr_$(APP_VERSION)/usr/local
	@mkdir onvifmgr_$(APP_VERSION)/usr/local/bin
	@cp "$(abs_builddir)/onvifmgr" onvifmgr_$(APP_VERSION)/usr/local/bin
```

In short, `onvifmgr` is simply copied into `/usr/local/bin`.

This is the lowest footprint a software could have...
